idl.md

```diff
@@ -15,7 +15,6 @@ dictionary StorageAccessTypes {
   boolean createObjectURL = false;
   boolean revokeObjectURL = false;
   boolean BroadcastChannel = false;
-  boolean SharedWorker = false;
```

Comment:
I'd be in favor of keeping this around and including a subtle note that this won't be part of the initial Chrome OT (this is a cross-browser proposal so we should keep it light but it's also important information for interested developers).